parsely/parselyandroid/ParselyTracker.java

```diff
@@ -331,10 +337,12 @@ protected ParselyTracker(String apikey, int flushInterval, String urlref, Contex
 
         this.apikey = apikey;
         this.uuidkey = "parsely-uuid";
+        this.adKey = null;
+        // get the adkey straight away on instantiation
+        new GetAdKey(c).execute();
         this.flushInterval = flushInterval;
         this.storageKey = "parsely-events.ser";
         //this.rootUrl = "http://10.0.2.2:5001/";  // emulator localhost
-        this.rootUrl = "http://srv.pixel.parsely.com/";
```

Comment:
Why is this removed? Looks like it's still important.

Reply:
it definitely is, not sure how I axed it and then missed it! adding back.